core/Date.php

```diff
@@ -605,7 +615,38 @@ public function subYear($n)
      * Returns a localized date string using the given template.
      * The template should contain tags that will be replaced with localized date strings.
      *
-     * Allowed tags include:
+     * @param string $template eg. `"MMM y"`
+     * @return string eg. `"Aug 2009"`
+     */
+    public function getLocalized($template)
+    {
+        $template = $this->replaceLegacyPlaceholders($template);
```

Comment:
It's only called when outputting some dates. But I guess we could check for % and only do the legacy stuff then